docs/tutorials/regression/bnn_vi_elbo.ipynb

```diff
@@ -329,30 +331,14 @@
     ")"
    ]
   },
-  {
-   "cell_type": "markdown",
-   "id": "08b3ef41",
-   "metadata": {},
-   "source": [
-    "## Prediction"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "963cfbb3",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "preds = bbp_model.predict_step(X_test)"
-   ]
-  },
   {
    "cell_type": "markdown",
    "id": "f9b45c30",
    "metadata": {},
    "source": [
-    "## Evaluate Predictions"
+    "## Evaluate Predictions\n",
+    "\n",
+    "The constructed Data Module contains two possible test variable. `X_test` are IID samples from the same noise distribution as the training data, while `X_gtext` are dense inputs from the underlying \"ground truth\" function without any noise that also extends the input range to either side, so we can visualize the method's UQ tendencies when extrapolating beyond the training data range. Thus, we will use `X_gtext` for visualization purposes, but use `X_test` to compute uncertainty and calibration metrics because we want to analyse how well the method has learned the noisy data distribution."
```

Comment:
@elcorto This is a draft for the proposed text, I would add to the notebooks to explain what variables we use for what and why. If we settle on one, I will add it to the other notebooks.